### DISCUSSION TITLE
Rendering on top of NativeControlHost

### DISCUSSION THREAD

**Discussion by jgcodes2020** [AvaloniaUI/Avalonia#12361] [Q&A]
Since NativeControlHost works using an external window, I would guess that it isn't possible to render above a NativeControlHost. Though I can't be completely sure. Is there any known way to render on top of a NativeControlHost?

**Comment by kekekeks** [comment]
You can place another native control as a child of your native control and render there. That is platform/control specific, so your mileage may vary.

**Reply by maxkatz6** [reply]
Or on some platforms (desktop only) you can put a popup on top of it.

**Comment by Coloryr** [comment]
> You can place another native control as a child of your native control and render there. That is platform/control specific, so your mileage may vary.

Can you provide some examples? I want to render something on NativeControlHost
